### PR TITLE
EOS-27256: IEM server integration with new event message changes

### PR DESCRIPTION
**Pull request by suryakumar1024**
Signed-off-by: suryakumar.kumaravelan <suryakumar.kumaravelan@seagate.com>

# Problem Statement
- Iem server fail to send and receive message since Event message api params are changed

# Design
-  Integrated Iem server with new event message api changes

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [x] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: NA
- [x] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: NA
- [x] Confirm Testing was performed with installed RPM [Y/N]:  Y

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [x] Is there a change in filename/package/module or signature [Y/N]: N
- [x] If yes for above point, Is a notification sent to all other cortx components [Y/N]: NA
- [ ] Jira is updated
- [x] Check if the description is clear and explained. 
- [x] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

**Review by rohit-k-dwivedi — APPROVED** [review]
LGTM

File: py-utils/src/utils/iem_framework/iem_server.py [hunk 1]
```diff
@@ -78,8 +90,11 @@ async def receive(request):
             f"{request.rel_url.query['component']}")
         try:
             cluster_conf = CortxConf.get_cluster_conf_path()
+            print(cluster_conf)
```
Comment (sachinpunadikar) [inline py-utils/src/utils/iem_framework/iem_server.py:93]: remove this line

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
work on comment

File: py-utils/src/utils/iem_framework/iem_server.py (hunk 1 above)
Comment (suryakumar1024) [reply py-utils/src/utils/iem_framework/iem_server.py]: Updated in recent patch

**Review by sachinpunadikar — APPROVED** [review]
+1

**Review by selvakumaar5496 — APPROVED** [review]

**Pull request merged**